src/test/java/lv/ctco/javaschool/goal/boundary/GoalApiTest.java

```diff
@@ -218,6 +220,18 @@ void testSaveGoalException() {
         assertThrows(InvalidGoalException.class, () -> goalApi.saveGoal(goalFormDto));
     }
 
+    @Test
+    void testCreateNewGoalInvalidDeadline() {
+        GoalFormDto goalFormDto = new GoalFormDto();
```

Comment:
It would be great if you would extract goalFromDto initialization and setup in to separate method